try {
    println("Hello world");
    throw "This is the error message";
    println("Fail");
}
catch (e) {
    println(e);
    println("OK");
}
